coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -396,6 +402,30 @@ def test_mv(self, matrix_shape, backend):
         TorchBaseTest.run_compare_torch((matrix, vector), model, backend=backend, input_as_shape=False)
 
 
+class TestCossim(TorchBaseTest):
+    
```

Comment:
https://pytorch.org/docs/stable/generated/torch.nn.CosineSimilarity.html
Please complete the unit test,
by parametrizing the `dim` and `eps` for this layer.
Please looks at other unit tests for reference

Reply:
Added tests for dim, eps and shapes, backends (mlprogram, fp16) excluded